ldes-server-port-ingest-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/rest/converters/IngestedModelConverter.java

```diff
@@ -54,7 +56,11 @@ public Model read(@NotNull Class<? extends Model> clazz, HttpInputMessage inputM
 						requireNonNull(inputMessage.getHeaders().getContentType()),
 						RdfFormatException.RdfFormatContext.INGEST
 				);
-		return RDFParser.source(inputMessage.getBody()).context(rdfModelConverter.getContext()).lang(lang).toModel();
+		Dataset dataset = RDFParser.source(inputMessage.getBody()).context(rdfModelConverter.getContext()).lang(lang).toDataset();
+		if (dataset.listModelNames().hasNext()) {
+			throw new IngestValidationException("Member can not contain named graphs");
+		}
+		return dataset.getDefaultModel();
```

Comment:
I am not a big fan of performing validation here, but I don't see where else it could happen (or the request body should be converted to a dataset instead of a model), so I am going to leave it like this for now